# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/WorkerDebugger.cpp
# Commit: 1e774bd7f365
# Full Hash: 1e774bd7f36525a03b6260b1cd7741bcf4e5f622
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-04-09 04:12:49
# Regressor Bug: 1899503
# File Overlap Count: 4
# Description:
#   Bug 1899503 - P3 PRemoteWorkerDebugger binding implementation. r=asuth
#   
#   This patch implements the PRemoteWorkerDebugger binding.
#   
#   We are trying to keep the same logic debugger registration/unregistration to minimize the difference between the local debugger(the debugger on the content process main thread)
# ==============================================================================

diff -r b003e26209cf -r 1e774bd7f365 dom/workers/WorkerDebugger.cpp
--- a/dom/workers/WorkerDebugger.cpp	Tue Apr 08 09:43:33 2025 +0000
+++ b/dom/workers/WorkerDebugger.cpp	Tue Apr 08 09:43:33 2025 +0000
@@ -209,6 +209,14 @@
 }
 
 NS_IMETHODIMP
+WorkerDebugger::GetIsRemote(bool* aResult) {
+  AssertIsOnMainThread();
+
+  *aResult = false;
+  return NS_OK;
+}
+
+NS_IMETHODIMP
 WorkerDebugger::GetIsInitialized(bool* aResult) {
   AssertIsOnMainThread();
 